### PR TITLE
rename functions per #65 update docs and tests

### PR DESCRIPTION
**Pull request by chendaniely**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,21 +11,27 @@
...
...pO30KCS5zdDEze29wYWNpdHk6MC4xODtmaWxsOnVybCgjU1ZHSURfMTFfKTt9Cgkuc3QxNHtvcGFjaXR5OjAuMzt9Cjwvc3R5bGU+CjxnIGlkPSJHcmF5X0xvZ28iPgo8L2c+CjxnIGlkPSJCbGFja19MZXR0ZXJzIj4KPC9nPgo8ZyBpZD0iQmx1ZV9HcmFkaWVudF9MZXR0ZXJzIj4KCTxnPgoKCQkJPGVsbGlwc2UgdHJhbnNmb3JtPSJtYXRyaXgoMC43MDcxIC0wLjcwNzEgMC43MDcxIDAuNzA3MSAtMTI3LjkyNjUgMzE3LjAzMTcpIiBjbGFzcz0ic3QwIiBjeD0iMzE4LjciIGN5PSIzMTIuOSIgcng9IjMwOS44IiByeT0iMzA5LjgiLz4KCQk8Zz4KCQkJPHBhdGggY2xhc3M9InN0MiIgZD0iTTQyNC43LDQxMS44aDMzLjZ2MjYuMWgtNTEuM0wzMjIsMzEwLjVoLTQ1LjN2MTAxLjNoNDQuM3YyNi4xSDIwOS41di0yNi4xaDM4LjNWMTg3LjNsLTM4LjMtNC43di0yNC43ICAgICBjMTQuNSwzLjMsMjcuMSw1LjYsNDIuOSw1LjZjMjMuOCwwLDQ4LjEtNS42LDcxLjktNS42YzQ2LjIsMCw4OS4xLDIxLDg5LjEsNzIuM2MwLDM5LjctMjMuOCw2NC45LTYwLjcsNzUuNkw0MjQuNyw0MTEuOHogICAgICBNMjc2LjcsMjg1LjNsMjQuMywwLjVjNTkuMywwLjksODIuMS0yMS45LDgyLjEtNTIuM2MwLTM1LjUtMjUuNy00OS41LTU4LjMtNDkuNWMtMTUuNCwwLTMxLjMsMS40LTQ4LjEsMy4zVjI4NS4zeiIvPgoJCTwvZz4KCTwvZz4KPC9nPgo8ZyBpZD0iV2hpdGVfTGV0dGVycyI+CjwvZz4KPGcgaWQ9IlJfQmFsbCI+CjwvZz4KPC9zdmc+)](https://community.rstudio.com/new-topic?title=&category_id=13&tags=grader&body=%0A%0A%0A%20%20--------%0A%20%20%0A%20%20%3Csup%3EReferred%20here%20by%20%60grader%60%27s%20README%3C/sup%3E%0A&u=barret)
```
Comment (schloerke) [inline README.md]: grader -> gradethis

File: R/gradethis_demo.R [hunk 2]
```diff
@@ -21,11 +21,12 @@
 #'
 #' @export
 #' @importFrom utils browseURL
-grading_demo <- function() {
-  grading_demo_path <- system.file("tutorials", "grading-demo/grading-demo.Rmd", package = "grader")
+gradethis_demo <- function() {
+  gradethis_demo_path <- system.file("tutorials", "grading-demo/grading-demo.Rmd",
```
Comment (schloerke) [inline R/gradethis_demo.R]: rename `grading-demo/grading-demo.Rmd` to `gradethis-demo/gradethis-demo.Rmd`

Comment (schloerke) [reply R/gradethis_demo.R]: ^^ The file and references to it throughout the package

File: R/grade_conditions.R [hunk 3]
```diff
@@ -32,25 +36,8 @@
 #' @seealso `test`
 #' @export
 #' @examples
-#' \dontrun{grading_demo()}
-#'
-#' example_function <- function(x){
```
Comment (schloerke) [inline R/grade_conditions.R]: didn't want to keep these?

Comment (chendaniely) [reply R/grade_conditions.R]: uhh... correct, I missed this during rebase

Comment (schloerke) [reply R/grade_conditions.R]: Ok. Keep them off or add them back if you’d like

**Pull request merged**